changelog.txt

```diff
@@ -0,0 +1,4 @@
+feature - Added --no-collapse flag to "firebase database:profile" to prevent collapsing of similiar path names in report.
+feature - Supported __session cookie in emulated hosting proxy functions.

```

Comment:
This is probably a fixed, not a feature.

    fixed - __session cookie is now properly emulated for Hosting function calls